src/lib.rs

```diff
@@ -279,29 +284,102 @@ fn handle_tls(
     exporter_builder: TonicExporterBuilder,
     url: &str,
     ca_cert_path: Option<String>,
```

Comment:
Yeah, it would be hard to apply that, unless we bolt on a custom exporter that we manage. The expectation would be that if the `ca_cert` is rotated, the process using `otel-lib` will have to be restarted.